### PR TITLE
fix navigation for non admin users

### PR DESCRIPTION
**Pull request by ann26**
This PR fixes the navigation for non admin users, such as:
- [x] fix widget bug when adding/editing organisation for non-admin users
- [x] fix bug when editing the course
- [x] non admin users is allowed to view pending organisation list page, also allowed to edit/delete the unapproved organisation that he was just created as long as he is the organisation owners. But only staff can approve the organisation
- [x] clean up navigation for non admin users who is organisation owner, he is now able to see the edit/delete button in the organisation list page
- [x] give permissions for project owner to edit/add

### Pending page for non admin users but an organisation owners
screenshot from 2017-07-31 16-00-11

### While staff can see everything
screenshot from 2017-07-31 16-04-30

### Organisation list page for non admin users but an organisation owners
screenshot from 2017-07-31 16-01-25

### CODE REVIEW

File: django_project/certification/views/course_type.py [hunk 1]
```diff
@@ -214,10 +214,7 @@ def get_queryset(self):
         """
 
         qs = CourseType.objects.all()
-        if self.request.user.is_staff:
-            return qs
-        else:
-            return qs.filter(creator=self.request.user)
+        return qs
```
Comment (timlinux) [inline django_project/certification/views/course_type.py]: Shouldn't we be filtering by organization admins? And for views above? I guess it is ok for any user to view course types but they should not be able to edit unless they are staff or org admin?

Comment (ann26) [inline django_project/certification/views/course_type.py]: It is filtered in the front end, so user won't see the edit buttons unless they are the organisation owners. Yes, currently any users can view the everything in organisation details, but won't be able to edit it (read only).
like this one:
screenshot from 2017-07-31 16-12-07
screenshot from 2017-07-31 16-12-17
screenshot from 2017-07-31 16-11-53
screenshot from 2017-07-31 16-12-26

Comment (ann26) [inline django_project/certification/views/course_type.py]: Or is it better to filter it by organisation owners too @timlinux ?

**Comment by timlinux** [comment]
@ann26 can you can confirm that an owner of one organization can not edit details of another organization (that they are not owner of)?

**Comment by ann26** [comment]
@timlinux yes, here is one of example, the owner of organisation ICL cannot edit another organisation that they are not owner of:
screenshot from 2017-08-01 01-06-49

**Comment by timlinux** [comment]
OK perfect thanks. If you resolve the merge conflict I will merge and deploy.

**Comment by ann26** [comment]
Done @timlinux :)

**Comment by timlinux** [comment]
Thanks @ann26!

**Pull request merged**